scripts/travis/start_elasticdl_job.sh

```diff
@@ -0,0 +1,56 @@
+JOB_TYPE=$1
+
+kubectl apply -f elasticdl/manifests/elasticdl-rbac.yaml
+
+if [[ "$JOB_TYPE" == "odps" ]] && \
+{ [[ "$ODPS_ACCESS_ID" == "" ]] || \
+[[ "$ODPS_ACCESS_KEY" == "" ]]; }; then
+    echo "Skipping ODPS related integration tests since \
+    either ODPS_ACCESS_ID or ODPS_ACCESS_KEY is not set"
+else
+    echo "Running ElasticDL job: ${JOB_TYPE}"
+    if [[ "$JOB_TYPE" == "odps" ]]; then
+        export MAXCOMPUTE_TABLE="odps_integration_build_"`
```

Comment:
In the next PR, it seems that we should move this code snippet into a new shell script file `scripts/travis/create_odps_table.sh`.